vespalib/src/vespa/vespalib/datastore/array_store.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Please use new copyright and year 2021.

Reply:
year is not mandated, makes it easier for later

Reply:
Fixed